exercises/practice/dnd-character/src/dnd_character.cr

```diff
@@ -0,0 +1,6 @@
+class DndCharacter
+  # Write your code for the 'DnD Character' exercise in this file.
+  def initialize(name : String = "Blandalf the Beige")
+    @name = name
+  end
```

Comment:
Looks like `@name` is unused and unneeded. My vote is to remove it to avoid confusion.

Reply:
I took a cue from [this bit](https://github.com/exercism/problem-specifications/blob/a57b3547b750d0f37af6897685fdbb81f2a5cf0b/exercises/dnd-character/canonical-data.json#L11-L12) in the `canonical-data.json` file:
 ```json
    ...
    "It may be fun to provide a 'name' property for characters",
    "that may or may not be randomly generated but not tested.",
    ...
 ```
 
 but happy to remove it too! 🙂

Reply:
Ah, I didn't notice that!

@meatball133 what are your thoughts?

Reply:
Let's stick with the name